src/sql/workbench/browser/modelComponents/iconUtils.ts

```diff
@@ -52,3 +52,8 @@ function getIconUri(iconPath: string | URI): URI {
 		return URI.revive(iconPath);
 	}
 }
+
+export function getIconKey(iconPath: IUserFriendlyIcon) {
+	const iconUri: URI = getLightIconUri(iconPath);
```

Comment:
what if the light icon is the same but dark is different, let's be safe here by considering both.